pyproject.toml

```diff
@@ -46,11 +46,11 @@ dev = [
     "pytest",
     "structlog",
     "python-logstash==0.4.6",
```

Comment:
Does this work with `python-logstash>=0.4.6`?

Reply:
Yes, it works. I will bump it up then